### ISSUE TITLE
As a SO/CE I want to create an accessible organizational chart

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1512]
## Context

Many sites use images for organizational charts and this is not accessible.  While we recommend that first a nested, unordered list should be used first, and then (optionally) followed by some sort of image, the presentation/display of this list could be improved.

## Possible Fix

1. Add a "tree" style for unordered lists. This works fine for just simple content, but is more difficult to work with in the WYSIWYG editor when content becomes more advanced, for example any content that would require a paragraph break.

![screen shot 2016-07-28 at 2 52 15 pm](https://user-images.githubusercontent.com/349713/30170925-b77cc874-93ad-11e7-8cc1-d2dffb69d7eb.jpg)

2. Create another way of organizing people that could be rendered in an organizational chart structure that could provide more advanced content rendering.  This could include adding references to person nodes, links to other sites, etc.

Creating something that could be managed similarly to the Drupal menu system (vertical + horizontal dragging) would be pretty neat.

https://codepen.io/kevincrafts/pen/ZyVNgW?editors=1100
